src/components/detail/ContentData.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="blog-content-data" v-html="data" />
+</template>
+<script setup>
+import { defineProps } from "vue"
+
+defineProps({
+  data: {
+    type: String,
+    required: true
+  }
+})
+</script>
+<style lang="scss">
+.blog-content-data {
+  flex: 65%;
+  padding: 0 2rem;
+
+  @media only screen and (max-width: 768px) {
+    flex: 100%;
+  }
+
+  p {
+    font-size: 1rem;
+    line-height: 1.6rem;
+  }
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  p,
+  pre,
+  ul {
```

Comment:
```suggestion
  h1, h2, h3, h4, h5, h6, p, pre, ul {
```
I prefer this but I am not sure yours might be the standard style.

Reply:
stylelint does not like that type of declarations.